### ISSUE TITLE
关于PR曲线

### ISSUE THREAD

**Issue by Marm7** [lartpang/ZoomNet#3]
想知道文中关于pr曲线和F曲线的绘制方法
我使用 sklearn.metrics 的 precision_recall_curve 函数
将数据集每张图的像素信息保存为 N*1 维数组再送入此函数中，但是对大型数据集来说速度非常慢
请问能否告知您的方法

**Comment by lartpang** [comment]
@Marm7 

I use my tool to plot these maps: https://github.com/lartpang/PySODEvalToolkit
And it is based another metric library implemented by me: https://github.com/lartpang/PySODMetrics

Maybe these codes can give you some inspiration: https://github.com/lartpang/PySODMetrics/blob/4aa253a59aff71507f92daf2dffe539c5c97ce46/py_sod_metrics/sod_metrics.py#L93
